realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/mongodb/MongoCollectionTest.kt

```diff
@@ -51,6 +54,9 @@ class MongoCollectionTest {
     @Before
     fun setUp() {
         Realm.init(InstrumentationRegistry.getInstrumentation().targetContext)
+
+        RealmLog.setLevel(LogLevel.DEBUG)
+
```

Comment:
I will also break down some of the tests into smaller, more meaningful cases in the upcoming commits.